packages/desktop-client/src/components/filters/CompactFiltersButton.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+import { Filter } from '../../icons/v1';
+import Button from '../common/Button';
+import View from '../common/View';
+
+type CompactFiltersButtonProps = {
+  onClick: (newValue) => void;
+};
+
+function CompactFiltersButton({ onClick }: CompactFiltersButtonProps) {
+  return (
+    <View>
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏I don't think we need to wrap it in `View` (but I might be wrong)